### ISSUE TITLE
[BUG] combining `sleep` with `at random` does not work (level 3)

### ISSUE THREAD

**Issue by juliabolt** [Felienne/hedy#2431]
**Describe the bug**
Also see issue #1938 with a similar problem. In this case however, when combining `sleep` with `at random`, in level 3, we get an error.

**Paste the Hedy code & level**
```
list is 1, 2, 3
print hi
sleep list at random
print bye
```

Level 3

**Expected behavior**
I would expect that it works, that `list at random` would be replaced by one of the elements, so a number, which would then determine how long `sleep` would happen.

**Comment by juliabolt** [comment]
Thanks @thjazi39 for bringing this to my attention

**Comment by thjazi39** [comment]
It surely has something to do with this issue: #1753

**Comment by Robernc** [comment]
Definitely. It seems like sleep has an issue handling variable/s and only works directly with integers.

**Comment by Liz433** [comment]
Hi! I would like to try and fix this issue!

**Comment by Felienne** [comment]
> Hi! I would like to try and fix this issue! Currently working on it if that is ok

Absolutely!

**Comment by juliabolt** [comment]
I tested this again, and now it works, so I think this issue has been solved?
